packages/eds-core-react/stories/playground/DataTable.tsx

```diff
@@ -112,7 +113,42 @@ const toCellValues = (data: Data[], columns: Column[]) =>
 
 const MenuButton = ({ row }: { row: string[] }) => {
   const [anchorEl, setAnchorEl] = useState<HTMLElement>()
+  const [focus, setFocus] = useState<MenuProps['focus']>(null)
   const isOpen = Boolean(anchorEl)
+
+  const openMenu = (focus: MenuProps['focus']) => {
+    setFocus(focus)
+  }
+  const closeMenu = () => {
+    setFocus(null)
+    setAnchorEl(null)
+  }
+
+  const onKeyPress = (e: React.KeyboardEvent<HTMLButtonElement>) => {
+    const { key } = e
+    e.preventDefault()
+    e.stopPropagation()
+    setAnchorEl(e.currentTarget)
```

Comment:
Wouldn't this open `Menu` on any keypress on `Button`? 😅 

Maybe setting the `anchorEl` is best done inside `openMenu` as its what we are semantically doing when we set it.

Reply:
Good call, thanks 👍